src/javascripts/Views/referencedByView.js

```diff
@@ -82,23 +70,19 @@ ngapp.controller('referencedByViewController', function($scope, $element, $timeo
 
     $scope.$watch('record', function(newValue, oldValue) {
         if (oldValue === newValue) return;
+        if (oldValue) $scope.releaseHandles(oldValue);
         if (!newValue) {
             $scope.view.label = 'Referenced By View';
             return;
         }
-        if (!xelib.IsMaster(newValue)) {           
-            $scope.record = xelib.GetMasterRecord(newValue);
-        } else {
-            $scope.view.label = xelib.Name(newValue);
-            $scope.focusedIndex = -1;
-            $scope.buildColumns();
-            $scope.buildTree();
-            $scope.$broadcast('recordChanged');
-            $timeout($scope.resolveElements, 100);
-        }
+        $scope.view.label = xelib.Name(newValue);
+        $scope.focusedIndex = -1;
+        $scope.buildColumns();
+        $scope.buildTree();
+        $timeout($scope.resolveElements, 100);
     });
 
-    $scope.showAddressBar = true;
+    $scope.showAddressBar = false;
```

Comment:
I don't think the referenced by view should have an address bar, so just nuke this line of code, the HTML for the address bar on this view, and any other javascript related to opening it.